Civi/Api4/PreferencesDate.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ */
+
+
+namespace Civi\Api4;
+
+/**
+ * PreferencesDate entity.
+ *
+ * @since 5.41
```

Comment:
@pradpnayak can you update this to be the correct version 5.69 I think it should be

Reply:
Hi @seamuslee001 missed that, pushed the change